app/lib/constants/visibility.rb

```diff
@@ -0,0 +1,4 @@
+module Constants::VisibilityTypes
+  PUBLIC = 'public'
+  PRIVATE = 'private'
```

Comment:
These should also ideally be numbers

```suggestion
  PUBLIC = 0
  PRIVATE = 1
```